File: Pattern 7/Pattern_7.py
class Solution:
    def printTriangle(self, n):
        for i in range(0,n):
            for j in range(n-1,i,-1):
                print(end= " ")
            for j in range(0,2*i+1):    
                print(end= "*");
            print()
        

File: Pattern 7/Pattern_7.cpp
class Solution {
  public:
    void printTriangle(int n) {
        
        for(int i =0; i<n; i++) {
            for(int j =n-1; j>i; j--) {
                cout<<" ";
            }
            for(int j =0; j<2*i+1; j++) {
                cout<<"*";
            }
            cout<<endl;
        }
        
    }
};